aptos-move/framework/aptos-framework/sources/delegation_pool.move

```diff
@@ -347,6 +368,13 @@ module aptos_framework::delegation_pool {
         voter: address,
     }
 
+    #[event]
+    struct SetBeneficiaryForOperator has drop, store {
```

Comment:
Do we have another event? If not, the suffix `ForOperator` seems unnecessary. And is it `of operator` or `for operator`?

Reply:
"ForOperator" specifies what it's for. Potentially, there can be beneficiaries for other types of accounts such as stakers.